{{cookiecutter.app_name}}/README.md

```diff
@@ -41,7 +41,7 @@ In general, before running shell commands, set the ``FLASK_APP`` and
 ``FLASK_DEBUG`` environment variables:
 
 ```sh
-    export FLASK_APP=/path/to/autoapp.py
```

Comment:
Can you make a separate heading for ENV Variables like this https://github.com/fulfilio/bifrost/blob/develop/README.md ?